INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 8, in index
    return render(request, 'map/index.html')
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: map/index.html
ERROR:django.request:Internal Server Error: /map/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 8, in index
    return render(request, 'map/index.html')
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: map/index.html
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /polls/log_coordinates/
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:root:Hello, world!
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:root:Hello, world!
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 44, in addFeature
    return HttpResponse(StageOneInstance.featureId)
AttributeError: 'StageOne' object has no attribute 'featureId'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:root:Hello, world!
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/step1Reset/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: step1Reset() takes 0 positional arguments but 1 was given
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/step1Reset/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: step1Reset() takes 0 positional arguments but 1 was given
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/step1Reset/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 332, in check_response
    raise ValueError(
ValueError: The view map.views.step1Reset didn't return an HttpResponse object. It returned None instead.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /ma
ERROR:django.request:Internal Server Error: /map/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 16, in index
    request.session['stage_one_instance'] = stage_one_instance.save_state()
AttributeError: 'StageOne' object has no attribute 'save_state'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 16, in index
    request.session['stage_one_instance'] = stage_one_instance.save_state()
AttributeError: 'StageOne' object has no attribute 'save_state'
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\middleware.py", line 59, in process_response
    request.session.save()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 80, in save
    return self.create()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 54, in create
    self.save(must_create=True)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 82, in save
    obj = self.create_model_instance(data)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 69, in create_model_instance
    session_data=self.encode(data),
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\base.py", line 94, in encode
    return signing.dumps(
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 152, in dumps
    return TimestampSigner(key=key, salt=salt).sign_object(
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 250, in sign_object
    data = serializer().dumps(obj)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 127, in dumps
    return json.dumps(obj, separators=(",", ":")).encode("latin-1")
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "json\encoder.py", line 179, in default
TypeError: Object of type StageOne is not JSON serializable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/step1Reset/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\middleware.py", line 59, in process_response
    request.session.save()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 80, in save
    return self.create()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 54, in create
    self.save(must_create=True)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 82, in save
    obj = self.create_model_instance(data)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\db.py", line 69, in create_model_instance
    session_data=self.encode(data),
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\contrib\sessions\backends\base.py", line 94, in encode
    return signing.dumps(
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 152, in dumps
    return TimestampSigner(key=key, salt=salt).sign_object(
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 250, in sign_object
    data = serializer().dumps(obj)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\signing.py", line 127, in dumps
    return json.dumps(obj, separators=(",", ":")).encode("latin-1")
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "json\encoder.py", line 179, in default
TypeError: Object of type StageOne is not JSON serializable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /map/getFeatures/
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/getFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in getFeature
    return JsonResponse(stage_one_instance.features)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 724, in __init__
    raise TypeError(
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False.
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/getFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in getFeature
    return JsonResponse(stage_one_instance.features)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 724, in __init__
    raise TypeError(
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False.
ERROR:django.request:Internal Server Error: /map/getFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in getFeature
    return JsonResponse(stage_one_instance.features)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 724, in __init__
    raise TypeError(
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False.
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/getFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in getFeature
    return JsonResponse(stage_one_instance.features)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 724, in __init__
    raise TypeError(
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False.
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 38, in addFeature
    featureType = request.Post.get('featureType')
AttributeError: 'WSGIRequest' object has no attribute 'Post'
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 38, in addFeature
    featureType = request.Post.get('featureType')
AttributeError: 'WSGIRequest' object has no attribute 'Post'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 38, in addFeature
    featureType = request.Post.get('featureType')
AttributeError: 'WSGIRequest' object has no attribute 'Post'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 38, in addFeature
    featureType = request.Post.get('featureType')
AttributeError: 'WSGIRequest' object has no attribute 'Post'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 42, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 42, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 42, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 42, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 42, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 19, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 43, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 19, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 43, in addFeature
    stage_one_instance.addFeature(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in addFeature
    self.features[feature] = data
TypeError: list indices must be integers or slices, not str
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:root:Hello, world!
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 39, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 18, in dataConvert
    for point in feature.points :
AttributeError: 'dict' object has no attribute 'points'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 37, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 17, in dataConvert
    return feature.points
AttributeError: 'dict' object has no attribute 'points'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 40, in addFeature
    stage_one_instance.addFeatureTLM(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 61, in addFeatureTLM
    coordinates = dataConvert(rawFeature, "TLM")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 24, in dataConvert
    "structureHeigh" : feature["points"][i]["structureHeigh"]["magnitude"]
KeyError: 'structureHeigh'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 45, in addFeature
    stage_one_instance.addFeatureDCS(featureId, request.POST.get('featureData'))
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 95, in addFeatureDCS
    coordinates = dataConvert(data, "DCS")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 48, in dataConvert
    for i in range(len(feature["points"])) :
TypeError: 'NoneType' object is not subscriptable
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 45, in addFeature
    stage_one_instance.addFeatureDCS(featureId, request.POST.get('featureData'))
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 95, in addFeatureDCS
    coordinates = dataConvert(data, "DCS")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 48, in dataConvert
    for i in range(len(feature["points"])) :
TypeError: 'NoneType' object is not subscriptable
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 45, in addFeature
    stage_one_instance.addFeatureDCS(featureId, request.POST.get('featureData'))
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 95, in addFeatureDCS
    coordinates = dataConvert(data, "DCS")
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 48, in dataConvert
    for i in range(len(feature["points"])) :
TypeError: 'NoneType' object is not subscriptable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 45, in addFeature
    stage_one_instance.addFeatureDCS(featureId, request.POST.get('featureData'))
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 91, in addFeatureDCS
    coordinates = dataConvert("DCS", data)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 46, in dataConvert
    for i in range(len(feature["points"])) :
TypeError: 'NoneType' object is not subscriptable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 47, in addFeature
    return JsonResponse({'success': True, 'features': stage_one_instance.features,'request' : request})
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 731, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
  File "json\encoder.py", line 179, in default
TypeError: Object of type WSGIRequest is not JSON serializable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 46, in addFeature
    return JsonResponse({'success': True, 'features': request})
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 731, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
  File "json\encoder.py", line 179, in default
TypeError: Object of type WSGIRequest is not JSON serializable
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 46, in addFeature
    return JsonResponse({'success': True, 'features': request})
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 731, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
  File "json\encoder.py", line 179, in default
TypeError: Object of type WSGIRequest is not JSON serializable
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 46, in addFeature
    return JsonResponse({'success': True, 'features': request})
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\response.py", line 731, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
  File "json\__init__.py", line 234, in dumps
  File "json\encoder.py", line 199, in encode
  File "json\encoder.py", line 257, in iterencode
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
  File "json\encoder.py", line 179, in default
TypeError: Object of type WSGIRequest is not JSON serializable
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in addFeature
    stage_one_instance.addFeatureDCS(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 91, in addFeatureDCS
    coordinates = dataConvert("DCS", data)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 49, in dataConvert
    "description": feature["points"][i]["description"],
KeyError: 'description'
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 52, in addFeature
    stage_one_instance.addFeatureDCS(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 91, in addFeatureDCS
    coordinates = dataConvert("DCS", data)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 49, in dataConvert
    "description": feature["points"][i]["description"],
KeyError: 'description'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/remFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 20, in remFeature
    data = json.loads(request.body)
  File "json\__init__.py", line 357, in loads
  File "json\decoder.py", line 337, in decode
  File "json\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
WARNING:django.request:Not Found: /favicon.ico
ERROR:django.request:Internal Server Error: /map/remFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 26, in remFeature
    stage_one_instance.remFeature(featureId)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 100, in remFeature
    self.features.pop(featureId)
KeyError: None
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/validateStepOne/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 78, in validateStepOne
    successParameters.continuous = True
AttributeError: 'dict' object has no attribute 'continuous'
ERROR:django.request:Internal Server Error: /map/validateStepOne/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 78, in validateStepOne
    successParameters.continuous = True
AttributeError: 'dict' object has no attribute 'continuous'
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
WARNING:django.request:Bad Request: /
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
WARNING:django.request:Bad Request: /favicon.ico
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
WARNING:django.request:Bad Request: /map
ERROR:django.security.DisallowedHost:Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineToolVenev\lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '172.16.4.57:8000'. You may need to add '172.16.4.57' to ALLOWED_HOSTS.
WARNING:django.request:Bad Request: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\PowerlineTool\settings.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\PowerlineTool\settings.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
ERROR:django.request:Internal Server Error: /map/addFeature/
Traceback (most recent call last):
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py", line 54, in addFeature
    stage_one_instance.addFeatureDCS(featureId, featureData)
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 106, in addFeatureDCS
    coordinates = dataConvert("DCS", data)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py", line 77, in dataConvert
    line_coordinates.append(feature["points"][i+1]["location"]["x"], feature["points"][i+1]["location"]["y"])
TypeError: list.append() takes exactly one argument (2 given)
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\Mapfunctions.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\urls.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\PowerlineTool\map\views.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\venv\Lib\site-packages\django\contrib\admin\decorators.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\venv\Lib\site-packages\django\contrib\messages\__init__.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:C:\Users\gerth\ITV Consult AG\itvdata - 3_Geschaeft\33_Projekte\BAZL\7000_DCS\5 - Operation\Tools\Python Script DCS\HL Read Find Format\NEW\venv\Lib\site-packages\django\contrib\sites\shortcuts.py changed, reloading.
INFO:django.utils.autoreload:Watching for file changes with StatReloader
INFO:django.utils.autoreload:Watching for file changes with StatReloader
WARNING:django.request:Not Found: /favicon.ico
